modules/installation-osp-verifying-cluster-status.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-verifying-cluster-status_{context}"]
+= Verifying cluster status
+
+To verify your {product-title} cluster's status during or after a deployment:
```

Comment:
A deployment is something specific in OCP. This should probably be "installation."